tst_generics_inheritance.py

```diff
@@ -0,0 +1,69 @@
+from pydantic import BaseModel
```

Comment:
Do we delete the whole file or switch to actual tests?

Reply:
Oh, sorry, it's my test file, will delete

Reply:
fixed